### PR TITLE
Issue 4 - Adding variables

### LINKED ISSUE #4: Add variables to variables.tf
Add use of variables so that the module is a little more flexible when imported. A prime example is https://github.com/fredrikhgrelland/terraform-nomad-minio/blob/master/variables.tf

### PR DESCRIPTION
**Pull request by claesgill**
Closes #4 

* Added input and output variables 
* Updated `hcl`- and `tf`-files accordingly

### CODE REVIEW

File: example/datastack/main.tf [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-  redash_admin_username = "admin"
-  redash_admin_password = "admin123"
-  redash_admin_email_id = "admin@mail.com"
+  postgres_service_name   = module.postgres.service_name
+  postgres_container_port = module.postgres.port
+  postgres_username       = module.postgres.username
+  postgres_password       = module.postgres.password
+  redash_admin_username   = "admin"
+  redash_admin_password   = "admin123"
+  redash_admin_email_id   = "admin@mail.com"
```
Comment (pdmthorsrud) [inline example/datastack/main.tf]: What about putting this into variables?

File: example/redash_one_node/main.tf [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
-  redash_admin_username = "admin"
-  redash_admin_password = "admin123"
-  redash_admin_email_id = "admin@mail.com"
+  postgres_service_name   = module.postgres.service_name
+  postgres_container_port = module.postgres.port
+  postgres_username       = module.postgres.username
+  postgres_password       = module.postgres.password
+  redash_admin_username   = "admin"
+  redash_admin_password   = "admin123"
+  redash_admin_email_id   = "admin@mail.com"
```
Comment (pdmthorsrud) [inline example/redash_one_node/main.tf]: same as above

**Comment by pdmthorsrud** [comment]
A headsup: this needs  to be updated with changes in master, which has all modules in the example split into separate files.

File: example/datastack/main.tf (hunk 1 above)
Comment (claesgill) [reply example/datastack/main.tf]: Sure!

**Review by pdmthorsrud — APPROVED** [review]
LGTM

**Pull request merged**